apps/aecore/src/aec_consensus_hc.erl

```diff
@@ -158,11 +159,31 @@ start_ae(StakersEncoded) ->
                 [{HCPubkey, HCPrivkey}]
             end,
             StakersEncoded),
-    StakersMap = maps:from_list(Stakers),
+    Pinners = lists:flatmap(
+            fun(#{<<"parent_chain_account">> := #{<<"pub">> := EncodedPubkey,
+                                                 <<"priv">> := EncodedPrivkey}
+                 }) ->
+                 {PCPubkey, PCPrivkey} = validate_keypair(EncodedPubkey, EncodedPrivkey),
+                 [{PCPubkey, PCPrivkey}]
+            end,
+            PinnersEncoded),
+    StakersMap = maps:from_list(lists:append(Stakers, Pinners)),
```

Comment:
I'm confused, aren't pinners key-pairs living on the parent chain? Why are they mixed with Stakers? or is `StakersMap` just a poor variable name?

Reply:
All the Stakers list is really doing is load the `aec_preset_keys` genserver, so I add the PC keys into that as well. The separate list is used to setup the mapping between Stakers and Pinners.

Reply:
And TBH, I just hooked in to the already running code and didn't change any names that were already there...

Reply:
I realised that was probably the case a few lines down 👍 - at some convenient time let's name it something more sensible.

Reply:
👍